RealEstate/settings/base.py

```diff
@@ -100,3 +100,4 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
 STATIC_URL = '/static/'
+LOGIN_REDIRECT_URL = '/'
```

Comment:
This setting accepts named URLs similar to reverse, so we can change this to 'home'.